actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -55,15 +63,27 @@ def call(env)
 
       response
     rescue Exception => exception
+      backtrace_cleaner = request.get_header('action_dispatch.backtrace_cleaner')
+      wrapper = ExceptionWrapper.new(backtrace_cleaner, exception)
+
+      invoke_interceptors(request, exception, wrapper)
       raise exception unless request.show_exceptions?
-      render_exception(request, exception)
+      render_exception(request, exception, wrapper)
```

Comment:
Actually, this breaks the render_exception interface, and plugin authors already depend on it. I'm going with the duplicated code approach. Yeah, I like DRY code, but sometimes the duplication is so much better than breaking interfaces or leaking abstractions.